### PR TITLE
Final touch: Todo list app

### PR DESCRIPTION
**Pull request by AsumVictor**
# Final touch: Todo list app
In this PR, I have added functionalities to function the todo list app work as expected
 
## Type of change
 * [ ] Added functions to add a new todo 
 * [ ] Added function to remove todo 
 * [ ] Added a function to edit to list
 * [ ] Added a function to display todo on page
 
 # Checklist:
 * [ ]  My code follows the style guidelines of this project
 * [ ] I have performed a self-review of my code
 * [ ] There is no linter error in my code
 * [ ]  My changes generate no new warnings

### CODE REVIEW

File: src/components/TodoForm.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  const [value, setValue] = useState('');
+  const [error, setError] = useState(false);
+
+  const todo = {
+    id: Math.ceil(Math.random() * 1000),
+    title: value,
+    createdAt: new Date(),
+    updatedAt: new Date(),
+    completed: false,
+  };
```
Comment (Dorcas126) [inline src/components/TodoForm.js]: You should use a unique string for each id. Random numbers from 1 to 1000 can repeat. There you suggest to use something like uuid to generate unique id.

**Review by Dorcas126 — CHANGES_REQUESTED** [review]
Hi @AsumVictor 
well done on your project👍, but you need to make some changes to make your project meet the requirements
ReallyReallyCloseBillMurrayGIF

# Highlights
Todo list display ✅
Can add todo list   ✅
App worked as expected  ✅
No linters errors  ✅

File: src/components/TodoForm.js (hunk 1 above)
Comment (AsumVictor) [reply src/components/TodoForm.js]: Okay. Great thank you

**Review by Dorcas126 — APPROVED** [review]
Hi @AsumVictor 
There's nothing to say, it's time to merge your project.
SvaRadioFmCongratsGIF

**Comment by AsumVictor** [comment]
Thank you very much!

**Pull request merged**